libraries/botframework-streaming/tests/NodeWebSocket.test.js

```diff
@@ -3,74 +3,68 @@ const { expect } = require('chai');
 const { FauxSock, TestRequest } = require('./helpers');
 const { randomBytes } = require('crypto');
 
-describe('NodeWebSocket', () => {
-    it('creates a new NodeWebSocket', () => {
-        const socket = new NodeWebSocket(new FauxSock);
+describe('NodeWebSocket', function () {
+    it('creates a new NodeWebSocket', function () {
+        const socket = new NodeWebSocket(new FauxSock());
         expect(socket).to.be.instanceOf(NodeWebSocket);
         expect(socket.close()).to.not.throw;
     });
 
-    it('requires a valid URL', () => {

```

Comment:
Note: This test was deleted because once "fixed" so that it waits for an expected error, the test doesn't actually throw one. Per @stevengum, and based on my own search, this error doesn't actually exist so the test is fine to delete.